05-policy-changes/policy-example.ipynb

```diff
@@ -0,0 +1,372 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    metadata = client.datastores.documents.metadata(datastore_id = datastore_id, document_id = uploaded_docs['fema_pappg-v4-updated-links_policy_6-1-2020.pdf'])
</pre><p>Make this loop over the list of ingested docs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ContextualAI/examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
ok, added a script for this